src/app/personsok/PersonsokSkjema.tsx

```diff
@@ -93,6 +93,18 @@ function PersonsokSkjema(props: Props) {
                     </section>
                 </SectionStyle>
                 <LenkeDrekV2 watch={form.watch} />
+                <a
+                    className="lenke"
+                    target={'_blank'}
+                    rel={'noopener noreferrer'}
+                    href={
+                        import.meta.env.PROD
+                            ? 'https://utbetalingsportalen.intern.nav.no'
+                            : 'https://utbetalingsportalen.intern.dev.nav.no'
```

Comment:
Hva tenker du om å putte URLene i miljøvariabler? For å samle config på ett sted

Reply:
skulle gjerne gjort det :sweat_smile:. Men da må (vil jeg) ha en mulighet for å sende miljøvariabler til frontenden i runtime. Per nå så ville vi måtte sette de build-time og da blir poenget litt borte.

Reply:
Men det er definitivt noe som har irritert lenge, og som jeg har planer om å legge til i https://github.com/navikt/modia-frontend